aws/provider.go

```diff
@@ -309,6 +309,7 @@ func Provider() terraform.ResourceProvider {
 			"aws_appsync_api_key":                              resourceAwsAppsyncApiKey(),
 			"aws_appsync_datasource":                           resourceAwsAppsyncDatasource(),
 			"aws_appsync_graphql_api":                          resourceAwsAppsyncGraphqlApi(),
+			"aws_appsync_schema":                               resourceAwsAppsyncSchema(),
```

Comment:
Can a AppSync Graph API have more than one schema? If not, instead of a new resource, it may make better sense as an attribute of the existing `aws_appsync_graphql_api` resource (unless self-references are necessary).

Reply:
There is only one schema per API.  I can look into adding it as an attribute instead of a separate resource.  This would move my StartSchemaCreation logic into the API create/update methods.